.github/known-licenses.txt

```diff
@@ -0,0 +1,22 @@
+Apache-1.0
+Apache-2.0
+BSD-2-Clause
+BSD-3-Clause
+curl
+GPL-1.0-or-later
+GPL-2.0-only
+GPL-3.0-only
+ISC
+MIT
+MPL-2.0
+PSF-2.0
+LGPL-2.0-or-later
+LGPL-2.1-only
+LGPL-2.0-only WITH OCaml-LGPL-linking-exception
+LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception
+LGPL-2.1-only WITH OCaml-LGPL-linking-exception
+LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception
+LGPL-3.0-only WITH OCaml-LGPL-linking-exception
+LGPL with OpenSSL linking exception
```

Comment:
We've looked and this is LGPL-2.1, for now lets keep it like this if it matches upstream